src/main/kotlin/net/atos/zac/zaken/ZakenService.kt

```diff
@@ -68,13 +70,21 @@ class ZakenService @Inject constructor(
                         explanation
                     )
                 }
-                indexeerService.indexeerDirect(
-                    zaak.uuid.toString(),
-                    ZoekObjectType.ZAAK,
-                    false
-                )
+                // unless OpenNotificaties is disabled, we don't need to do any indexing here,
```

Comment:
I don't think we should do this to be honest: have code that runs if we have notifications disabled. We should always use notifications in any circumstance I think. That is key functionality of ZAC. If we need to fake OpenNotifications locally or in our test we can 'simply' call our ZAC notifications endpoint directly, like we already do in our integration tests. I think adding this 'local development situation only' code is not the way we want to go. It adds more complexity to our code base, it is code that will never be used in real life, and it makes testing this code harder.

Reply:
That's fair. It might be a good idea to add something in the developer documentation, though.

Reply:
I agree. I think we do already have something on this topic? Not too sure. We should also be careful not to write too much docs because the maintenance quickly becomes an issue. It's already quite hard..